apps/entity-service/src/entities/processors/entity-processor.ts

```diff
@@ -61,7 +62,8 @@ export abstract class EntityProcessor<
   ModelType extends EntityModel,
   LightDto extends EntityDto,
   FullDto extends EntityDto,
-  UpdateDto extends EntityUpdateData
+  UpdateDto extends EntityUpdateData,
+  CreateDto extends EntityCreateData
```

Comment:
This should be optional so you don't have to provide it for every processor. Probably something like this:
```
CreateDto extends EntityCreateData = EntityCreateAttributes
```
That might cause some type errors; I'd need to play around to be sure. Let me know if you have trouble making this work.